### ISSUE TITLE
text with ampersand 'jumps'

### ISSUE THREAD

**Issue by evdama** [henriquehbr/svelte-typewriter#31]
If you put a `&` in your text for example then you get a jumping left/right effect which is bad

```html
<script>
  import Typewriter from 'svelte-typewriter'
</script>

<Typewriter loop interval={ 100 }>
  <div>normal text doesn't jump</div>
  <div>text with ambersand & then some more</div>         // <--- jumping effect when processing &
</Typewriter>
```

Repl to reproduce https://svelte.dev/repl/5698397692c54c2bb5fd03f5ea6ca8cc?version=3.31.0

**Comment by henriquehbr** [comment]
Indeed, just checked the repl, at the moment i'm solving conflicts caused by breaking changes on the dependencies, but i'll look into it as soon as possible

Many thanks for reporting this!

**Comment by henriquehbr** [comment]
A fix for this bug has been released on [`2.4.4`](https://github.com/henriquehbr/svelte-typewriter/blob/v2.4.4/CHANGELOG.md#v244)

**Comment by evdama** [comment]
Excellent, I just checked and yes, it's fixed!
Thanks a lot for your time and effort... it's much appreciated for this awesome package 👍
